engine/src/main/scala/cromwell/services/ServiceRegistryActor.scala

```diff
@@ -0,0 +1,46 @@
+package cromwell.services
+
+import akka.actor.{Actor, ActorRef, Props}
+import com.typesafe.config.{ConfigFactory, Config, ConfigObject}
+import cromwell.engine.db.DataAccess
+import cromwell.services.ServiceRegistryActor._
+import lenthall.config.ScalaConfig._
+
+import scala.collection.JavaConverters._
+
+object ServiceRegistryActor {
+  case class Send(serviceName: String, message: Any)
```

Comment:
What about having a `ServiceMessage` (or somesuch) type and having `message` accept that. Clients of the service registry would have access to it and could extend as appropriate.